packages/jest-jasmine2/src/index.js

```diff
@@ -85,12 +92,24 @@ async function jasmine2(
     if (config.resetMocks) {
       runtime.resetAllMocks();
 
+      if (!environment) {
+        throw new ReferenceError(
```

Comment:
Is there any way that this can actually happen? How about just doing `if (environment && config.timers === 'fake') { … } ?